src/sentry/tasks/statistical_detectors.py

```diff
@@ -153,20 +158,36 @@ def detect_function_trends(project_ids: List[int], start: datetime, *args, **kwa
     max_retries=0,
 )
 def detect_function_change_points(
-    functions: List[Tuple[int, str | int]], start: datetime, *args, **kwargs
+    functions_list: List[Tuple[int, int]], start: datetime, *args, **kwargs
 ) -> None:
-    for project_id, function_id in functions:
+    breakpoint_count = 0
+
+    breakpoints = _detect_function_change_points(functions_list, start)
+
+    for entry in breakpoints:
+        breakpoint_count += 1
+
         with sentry_sdk.push_scope() as scope:
-            scope.set_tag("regressed_project_id", project_id)
-            scope.set_tag("regressed_function_id", function_id)
+            scope.set_tag("regressed_project_id", entry["project"])
+            # the service was originally meant for transactions so this
+            # naming is a result of this
+            scope.set_tag("regressed_function_id", entry["transaction"])
             scope.set_context(
                 "statistical_detectors",
                 {
+                    **entry,
                     "timestamp": start.isoformat(),
+                    "breakpoint": datetime.fromtimestamp(entry["breakpoint"]),
                 },
             )
             sentry_sdk.capture_message("Potential Regression")
 
+    metrics.incr(
+        "statistical_detectors.breakpoint.functions",
+        amount=breakpoint_count,
+        sample_rate=1.0,
+    )
```

Comment:
do you think it also makes sense to record a percentage of timeseries that actually had breakpoints

Reply:
I think we can derive the percentage by dividing this metric with the existing metrics so just having this count will suffice.

Reply:
👍